relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkg.py

```diff
@@ -388,9 +388,9 @@ def update():
     update_licenses()
 
 
-"""
+'''
```

Comment:
This looks like an outdated `blue`. They changed their policy on quotes in August.

Reply:
oh

Reply:
indeed.